enclave-manager/web/src/client/enclaveManager/KurtosisClient.ts

```diff
@@ -73,11 +79,11 @@ export abstract class KurtosisClient {
       const apicInfo = enclave.apiContainerInfo;
       assertDefined(
         apicInfo,
-        `Cannot listFilesArtifactNamesAndUuids because the passed enclave '${enclave.name}' does not have apicInfo`,
+        `Cannot listFilesArtifactNamesAndUuids because the passed enclave '${enclave.name}' does not have apicInfo`
       );
       const request = new GetListFilesArtifactNamesAndUuidsRequest({
         apicIpAddress: apicInfo.bridgeIpAddress,
-        apicPort: apicInfo.grpcPortInsideEnclave,
+        apicPort: apicInfo.grpcPortInsideEnclave
```

Comment:
wonder if our prettiers aren't configured consistently - this change is just a formatting change. Could you run `yarn prettier:fix` and verify whether you keep this change.

Reply:
We should add an ignore for protobuf generated files as it also formats those guys.

Reply:
done